docs/concepts/alert_activations.ipynb

```diff
@@ -0,0 +1,550 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # to get activations for specific alert config
</pre><p>change to:</p>

<pre class="ql-syntax" spellcheck="false"># list alert activations for a specific alert config
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/6881/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>